test/failing/dune

```diff
@@ -0,0 +1,11 @@
+(rule
```

Comment:
I wonder if it would be better to generate both the failing and passing rules from the same script ? On the other hand, this tests may be removed or implemented differently if we need them in the future.

Reply:
I considered it, but it didn't seem worth the hassle right now (given there is only one failing test).

Reply:
Works for me.